puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/tfm-ror51-cleanup-copr.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- job:
+    name: tfm-ror51-cleanup-copr
+    project-type: workflow
+    logrotate:
```

Comment:
you configure `buildDiscarder` in he groovy file, so I'd think this is not needed here anymore?